lib/theme/hs_theme.dart

```diff
@@ -1,15 +1,20 @@
 import 'package:flutter/material.dart';
 import 'package:hatspace/gen/fonts.gen.dart';
+import 'package:hatspace/theme/hs_button_theme.dart';
 
 part 'hs_color.dart';
 part 'hs_text_theme.dart';
 
 ThemeData lightThemeData = ThemeData(
-    scaffoldBackgroundColor: colorScheme.background,
-    textTheme: textTheme,
-    colorScheme: colorScheme,
-    fontFamily: FontFamily.beVietnamPro,
-    dividerTheme: const DividerThemeData(
-      color: HSColor.neutral3,
-      thickness: 1.0,
-    ));
+  scaffoldBackgroundColor: colorScheme.background,
+  textTheme: textTheme,
+  colorScheme: colorScheme,
+  elevatedButtonTheme: primaryButtonTheme,
+  outlinedButtonTheme: secondaryButtonTheme,
+  textButtonTheme: textOnlyButtonTheme,
+  fontFamily: FontFamily.beVietnamPro,
```

Comment:
From what I've seen from the designs, I've only ever seen Manrope font, so I'm not sure about this. I'll just set it to Manrope for now if that's ok, since it's the more widely used one